api/grpc/server.go

```diff
@@ -666,6 +663,26 @@ func toGrpcBalInfo(src perun.BalInfo) *pb.BalInfo {
 	}
 }
 
+// toGrpcChsInfo is a helper function to convert slice of ChInfo struct defined in perun-node
```

Comment:
Once you implement the below comment, these functions will not be needed. So you can remove these and use `toGrpcPayChsInfo`which is already defined.